bundles/org.openhab.binding.dominoswiss/src/main/java/org/openhab/binding/dominoswiss/internal/blindHandler.java

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.dominoswiss.internal;
+
+import static org.openhab.binding.dominoswiss.internal.dominoswissBindingConstants.*;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.ThingStatusInfo;
+import org.openhab.core.thing.binding.BaseThingHandler;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.RefreshType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link BlindHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Frieso Aeschbacher - Initial contribution
+ */
+@NonNullByDefault
+public class blindHandler extends BaseThingHandler {
+
+    /**
+     * The {@link BlindHandler} class defines common constants, which are
+     * used across the whole binding.
+     *
+     * @author Frieso Aeschbacher - Initial contribution
+     */
+
+    private Logger logger = LoggerFactory.getLogger(blindHandler.class);
+
+    private @Nullable eGateHandler dominoswissHandler;
+
+    private String id = "";
+
+    public blindHandler(Thing thing) {
+        super(thing);
+    }
+
+    @SuppressWarnings("null")
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        logger.debug("Blind got command: {} and ChannelUID: {} ", command.toFullString(),
+                channelUID.getIdWithoutGroup());
+        try {
+            dominoswissHandler = (eGateHandler) getBridge().getHandler();
+        } catch (Exception e) {
+            logger.error("Could not get Bridge ", e);
+        }
+
+        if (dominoswissHandler == null) {
+            logger.info("Blind thing {} has no server configured, ignoring command: {}", getThing().getUID(), command);
+            return;
+        }
+
+        String id = getConfigAs(blindConfig.class).id;
+
+        // Some of the code below is not designed to handle REFRESH
+        if (command == RefreshType.REFRESH) {
+            return;
+        }
+        switch (channelUID.getIdWithoutGroup()) {
+            case CHANNEL_PULSEUP:
+                if (command instanceof Number) {
+                    dominoswissHandler.pulseUp(id);
```

Comment:
You could store `dominoswissHandler` to a local variable and do a null check on that to avoid the compiler warning.